library/config.py

```diff
@@ -78,6 +79,11 @@ def version_today(self) -> str:
     @property
     def compute(self) -> dict:
         """based on given yml file, compute the configuration"""
+
+        # Validate unparsed, unrendered file
+        v = Validator(self.parsed_unrendered_template)
+        v()
```

Comment:
can we call it validator? instead of v, we want to be verbose and specific

Reply:
`Validator(self.parsed_unrendered_template)()` you can also do this

Reply:
I agree, will change